src/libpriv/rpmostree-core.cxx

```diff
@@ -667,6 +667,19 @@ enable_repos (RpmOstreeContext  *context,
   return TRUE;
 }
 
+void 
+rpmostree_context_set_treespec (RpmOstreeContext *self, RpmOstreeTreespec *treespec)
+{
```

Comment:
Should we add a `g_assert (!self->spec)` here to make sure this is only called once?

Reply:
Done.